### PR TITLE
Update OAuth For Okta scope list with policy scope and user scope functionality

### PR DESCRIPTION
**Pull request by danielzeller-okta**
## Description:
- **What's changed?**
Update Scope list to reflect current scopes & functionality
- **Is this PR related to a Monolith release?**
Yes,  2020.02.0 

### Resolves:

* [OKTA-254864](https://oktainc.atlassian.net/browse/OKTA-254864)

## Reviewers:
@jameszhang-okta @larsjohansen-okta @motikagan-okta

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md [hunk 1]
```diff
@@ -26,10 +26,12 @@ The following table shows the scopes that are currently available as a part of E
 | `okta.inlineHooks.manage`| Allows the app to create and manage Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
 | `okta.inlineHooks.read` | Allows the app to read information about Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
 | `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
+| `okta.policies.manage`   | Allows the app to manage policies in your Okta organization             | [Policy API](/docs/reference/api/policy/) |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md]: `Allows the app to manage policies in your Okta organization ` -> `Allows the app to create and manage policies in your Okta organization `

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md [hunk 2]
```diff
@@ -26,10 +26,12 @@ The following table shows the scopes that are currently available as a part of E
...
+| `okta.policies.read`     | Allows the app to read information about  policies in your Okta organization | [Policy API](/docs/reference/api/policy/) |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md]: double spacing `about  policies` -> `about policies`

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md (hunk 1 above)
Comment (jameszhang-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md]: I believe I have updated the developer doc regarding okta.policies.manage and okta.policies.read in another PR https://github.com/okta/okta-developer-docs/pull/916. Something I missed here?

Comment (danielzeller-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md]: Ooops.. I just hadn't pulled recently enough! I'll remove these lines 😄

**Pull request merged**